packages/application-shell/src/components/project-container/project-container.js

```diff
@@ -108,42 +116,38 @@ export class ProjectContainer extends React.Component {
     if (this.props.user && this.props.user.projects.total === 0)
       return <Redirect to="/logout?reason=no-projects" />;
 
-    // A trial expire notification should be displayed from 2 weeks before the project expires
-    const minDaysToDisplayNotification = 14;
-    const maxDaysToDisplayNotification = 0;
-
     return (
-      <FetchProject projectKey={this.props.match.params.projectKey}>
-        {({ isLoading: isProjectLoading, project }) => {
-          // TODO: do something if there is an `error`?
-          if (isProjectLoading) return <ApplicationLoader />;
-          if (!project) return <ProjectNotFound />;
-          if (project.suspension && project.suspension.isActive)
-            return (
-              <ProjectSuspended
-                isTemporary={
-                  project.suspension.reason ===
-                  SUSPENSION_REASONS.TEMPORARY_MAINTENANCE
-                }
-              />
-            );
-          if (project.expiry && project.expiry.isActive)
-            return <ProjectExpired />;
+      <React.Suspense fallback={<ApplicationLoader />}>
```

Comment:
This diff is because of the indentation for adding `<React.Suspense>` to the tree